wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/CommandGroup.java

```diff
@@ -34,14 +34,14 @@
   /**
    * The commands in this group (stored in entries).
    */
-  private final Vector m_commands = new Vector();
+  private final List<Entry> m_commands = new ArrayList<>();
```

Comment:
Hmm. I know that command-based was specifically created to avoid threading, but I feel like teams might be doing weird things (as they always do). At the very least, we need to document that commands aren't threadsafe.